src/wazuh_modules/wm_office365.c

```diff
@@ -140,16 +149,33 @@ void * wm_office365_main(wm_office365* office365_config) {
         mtinfo(WM_OFFICE365_LOGTAG, "Module Office365 disabled.");
     }
 
+#ifdef WIN32
+    return 0;
+#else
     return NULL;
+#endif
 }
 
+#ifdef WIN32
+STATIC DWORD WINAPI wm_office365_destroy(void *office365_config) {
+    wm_office365 *office365_config_ptr = (wm_office365 *)office365_config;
+
+    mtinfo(WM_OFFICE365_LOGTAG, "Module Office365 finished.");
+    wm_office365_auth_destroy(office365_config_ptr->auth);
+    wm_office365_subscription_destroy(office365_config_ptr->subscription);
+    wm_office365_fail_destroy(office365_config_ptr->fails);
+    os_free(office365_config_ptr);
+    return 0;
```

Comment:
this could be summarized with a #ifdef WIN32 on the return to avoid the entire duplication of the function.